com.deflemask.DefleMask.appdata.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright 2019 Karim Vergnes <thesola10@bobile.fr>
+  -->
+<application>
```

Comment:
Run appstream-util upgarde on this file.

Reply:
There is no usable donation link, bugtracker, or translation link. Can I omit them?
Also who is `update_contact` supposed to refer to? Me, or the upstream devs?

Reply:
The person in charge of updating the package unless upstream is involved

Reply:
never mind, I figured out the `update_contact` by (drumroll) RTFM.